scout/load/all.py

```diff
@@ -102,16 +102,12 @@ def load_region(adapter, case_id, hgnc_id=None, chrom=None, start=None, end=None
     vcf_str_file = case_obj["vcf_files"].get("vcf_str")
     if vcf_str_file:
         LOG.info(
-            "Load clinical STR variants for case: {0} region: chr {1}, "
-            "start {2}, end {3}".format(case_obj["_id"], chrom, start, end)
+            "Load all clinical STR variants for case: {0}."
         )
         adapter.load_variants(
             case_obj=case_obj,
             variant_type="clinical",
             category="str",
-            chrom=chrom,
```

Comment:
Should work conceptually. But better test it first!

Reply:
Now you want to test.. Tsssk.. 😉

Reply:
Ok, on hasta stage!